qtfred/src/ui/widgets/sexp_tree.cpp

```diff
@@ -146,26 +146,28 @@ SCP_vector<SCP_string> SexpTreeEditorInterface::getMessages() {
 
 	return list;
 }
-SCP_vector<SCP_string> SexpTreeEditorInterface::getMissionGoals(const SCP_string&  /*reference_name*/) {
-	SCP_vector<SCP_string> list;
+QStringList SexpTreeEditorInterface::getMissionGoals(const QString&  /*reference_name*/) {
```

Comment:
While I think that switching from `SCP_*` data structures to Qt native ones was a good call :+1: I wonder if it'd be better to do these changes in a separate PR.

Reply:
These methods were not previously referenced by any other file, and indeed have no use outside of campaign creation. I felt they were simply ported from FRED without too much consideration, so I changed them.

Reply:
Ok, that sounds fair. Thanks for explaining.